CHANGELOG.rst

```diff
@@ -46,7 +45,7 @@ Minor Changes
 - zos_job_submit - The architecture changed such that the entire modules execution time now captured in the duration time which includes job submission and log collection. If a job does not return by the default 10 sec 'wait_time_s' value, it can be increased up to 86400 seconds. (https://github.com/ansible-collections/ibm_zos_core/issues/389).
 - zos_job_submit - behavior changed when a volume is defined in the module options such that it will catalog the data set if it is not cataloged and submit the job. In the past, the function did not catalog the data set and instead performed I/O operations and then submitted the job. This behavior aligns to other module behaviors and reduces the possibility to encounter a permissions issue. (https://github.com/ansible-collections/ibm_zos_core/issues/389).
 - zos_job_submit - was updated to include an additional error code condition JCLERR. (https://github.com/ansible-collections/ibm_zos_core/pull/312)
-- zos_lineinfile - updates the module with a new option named tmp_hlq. This allows for a user to specify the data set high level qualifier (HLQ) used in any temporary data set created by the module. Often, the defaults are not permitted on systems, this provides a way to override the defaults. (https://github.com/ansible-collections/ibm_zos_core/pull/341).
+- zos_lineinfile- updates the module with a new option named tmp_hlq. This allows for a user to specify the data set high level qualifier (HLQ) used in any temporary data set created by the module. Often, the defaults are not permitted on systems, this provides a way to override the defaults. (https://github.com/ansible-collections/ibm_zos_core/pull/341).
```

Comment:
I think this "- zos_lineinfile-" won't render the module name in color correctly like displayed in https://ibm.github.io/z_ansible_collections_doc/ibm_zos_core/docs/source/release_notes.html .

Reply:
Was checking a file below and this is not the one rendered into that page. I guess there won't be issues.

Reply:
Good catch for preparing for the future when can figure out how to use the release notes without a edit but as you point out its a NOOP because we authored the release notes ourselves, thus it was written as:
```
- ``zos_lineinfile`` - updates the module with a new option named tmp_hlq. This allows for a user to specify the data set high level qualifier (HLQ) used in any temporary data set created by the module. Often, the defaults are not permitted on systems, this provides a way to override the defaults.
```
But i did update it, looks like this came from fragement 341 which is no longer our branch , we must have flushed it in 1.5.0-beta.1, but i did push a change.

Reply:
Commit [here](https://github.com/ansible-collections/ibm_zos_core/pull/752/commits/1ea7be263f4970ea401881a7fc38be6ceac9387b) should resolve the conversation